EpiAware/src/EpiAwareUtils/turing-methods.jl

```diff
@@ -15,13 +15,14 @@ A `DynamicPPPL.Model` object.
         y_t, time_steps, epi_model::AbstractTuringEpiModel;
         latent_model::AbstractTuringLatentModel, observation_model::AbstractTuringObservationModel)
     # Latent process
-    @submodel Z_t, latent_model_aux = generate_latent(latent_model, time_steps)
+    @submodel prefix="latent" Z_t, latent_model_aux=generate_latent(
+        latent_model, time_steps)
 
     # Transform into infections
     @submodel I_t = generate_latent_infs(epi_model, Z_t)
 
     # Predictive distribution of ascertained cases
-    @submodel generated_y_t, generated_y_t_aux = generate_observations(
+    @submodel prefix="obs" generated_y_t, generated_y_t_aux=generate_observations(
```

Comment:
why have we changed to having no spaces around `=`?

Reply:
I've gone with the form of argument input described [here](https://turinglang.org/DynamicPPL.jl/stable/api/#DynamicPPL.@submodel). I haven't tested alternatives, but I expect that the `@submodel` macro is looking for a pattern of string input `prefix=...` with no spaces.

Another alternative is `@submodel(string_expr, expr)` rather than using spaces but thats also a style deviation from Julia/our norm.

Reply:
PS I'm not sure why JuliaFormatter got rid of the second set of whitespaces around `=`

Reply:
Okay no worries happy to listen to the formatting gods on this one